doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -229,11 +229,19 @@ nRF Desktop
   * The :ref:`nrf_desktop_dfu_lock`.
     The utility provides synchronization mechanism for accessing the DFU flash.
     It is especially useful for application configurations that support more than one DFU method.
+  * The :ref:`nrf_desktop_smp_custom` that you can enable with the :ref:`CONFIG_DESKTOP_DFU_MCUMGR_ENABLE <config_desktop_app_options>` option.
+    The module integrates the :ref:`nrf_desktop_dfu_lock` for synchronizing flash access with other DFU methods.
+    Use :ref:`CONFIG_DESKTOP_DFU_LOCK <config_desktop_app_options>` option to enable this feature.
 
 * Updated:
 
   * The :ref:`nrf_desktop_dfu` to integrate the :ref:`nrf_desktop_dfu_lock` for synchronizing flash access with other DFU methods.
     Use :ref:`CONFIG_DESKTOP_DFU_LOCK <config_desktop_app_options>` option to enable this feature.
+  * The following application configurations to use the :ref:`nrf_desktop_smp_custom`:
+      * Fast Pair configuration (:file:`prj_fast_pair.conf`) for the nRF52840 DK target.
+      * Fast Pair configuration (:file:`prj_fast_pair.conf`) for the nRF52840 Gaming Mouse target.
+      * Fast Pair release configuration (:file:`prj_release_fast_pair.conf`) for the nRF52 Keyboard target.
+      * MCUboot SMP configuration (:file:`prj_mcuboot_smp.conf`) for the nRF52840 DK target.
```

Comment:
You can mention only the prj_mcuboot_smp,conf for nRF52840DK. The Fast Pair configurations are already altogether mentioned here: https://github.com/nrfconnect/sdk-nrf/pull/11182